src/printer-python.js

```diff
@@ -0,0 +1,538 @@
+"use strict";
+
+const docBuilders = require("./doc-builders");
+const concat = docBuilders.concat;
+const join = docBuilders.join;
+const hardline = docBuilders.hardline;
+const line = docBuilders.line;
+const softline = docBuilders.softline;
+const group = docBuilders.group;
+const indent = docBuilders.indent;
+const ifBreak = docBuilders.ifBreak;
+
+function printArguments(print, path, argsKey, defaultsKey) {
+  const n = path.getValue();
+
+  // python AST represent arguments and default
+  // value in two different lists, so we grab
+  // the list of the arguments and the list of
+  // default values and we merge them together and sort
+  // them by column. Then we iterated one by one and
+  // if the next element is a default value we merge it with
+  // the current one
+
+  const merge = [...n[argsKey], ...n[defaultsKey]].sort(
```

Comment:
Node v4 doesn't support the spread operator, so you can use `n[args.key].concat(n[defaultsKey])`